README.md

```diff
@@ -188,3 +188,24 @@ linkchecker --check-extern target/*.html
 ```
 
 If you remove the command line switch `--check-extern` only local links are checked. 
+
+## Continuous Integration
+
+This project contains a CI setup for Jenkins. The build uses docker to provide a stable environment for its execution. So Jenkins needs to be able to execute docker commands.
+The build process itself is defined within `Jenkinsfile` and consists of these stages:
+- checkout : Get the source from repo
+- build docker : Create the docker image to be used during build execution (only once if it doesn't exist locally)
+- render html : render a html file from sources
+- render pdf : render a pdf file from source
+
+The rendering of html, pdf,... is controlled with maven profiles (render-html, render-pdf). To enable other/ additional formats new profiles can be defined within pom.xml.
```

Comment:
"controlled by Maven profiles"

set `render-html`, `render-pdf` and `pom.xml` in monospace (reverse single quotes)